app/assets/images/payment_icons/cash.svg

```diff
@@ -1 +1,2 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-cash" width="38" height="24" role="img"><title id="pi-cash">Cash</title><path fill="#98C74C" d="M38 21a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3h32a3 3 0 0 1 3 3v18z"/><circle opacity=".2" cx="19" cy="12" r="8"/><circle opacity=".2" cx="32" cy="12" r="2"/><circle opacity=".2" cx="6" cy="12" r="2"/><path fill="#fff" d="M22.113 13.897c0-1.573-1.271-2.067-2.518-2.42V8.763c.65.118 1.218.588 1.68 1.105l.71-.797c-.578-.609-1.424-1.128-2.39-1.246V6.83h-1.034v.973c-1.408.154-2.413 1.036-2.413 2.275 0 1.461 1.2 1.907 2.413 2.244v3.024c-.898-.145-1.608-.759-2.053-1.277l-.62.945c.597.673 1.504 1.151 2.674 1.266v.89h1.034v-.881c1.819-.186 2.517-1.304 2.517-2.392zm-4.974-3.948c0-.684.627-1.11 1.422-1.223v2.459c-.797-.133-1.422-.596-1.422-1.236zm2.456 5.426v-2.762c.813.254 1.451.59 1.451 1.359 0 .565-.364 1.246-1.451 1.403z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-{cash}">
+    <title id="pi-cash">Cash</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="M16.633 9.778c0-.611.556-1.193 1.617-1.384v2.794c-1.193-.296-1.617-.788-1.617-1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="a"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

Comment:
```suggestion
    <title id="pi-cash">Cash</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="M16.633 9.778c0-.611.556-1.193 1.617-1.384v2.794c-1.193-.296-1.617-.788-1.617-1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" /><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" /><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" /></g><defs><clipPath id="a"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

The `style` tag is redundant and p3 colors are not supported by all browsers. The `fill` is enough.

Reply:
Is the stoke opacity intentional?